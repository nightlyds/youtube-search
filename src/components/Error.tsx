import React from "react";

const Error: React.FC = () => {
    return (
        <div className="error">
            <span className="error-message">You caught the error :(</span>
        </div>
    );
};

export default Error;
