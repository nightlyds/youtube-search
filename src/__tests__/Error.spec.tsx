import React from "react";
import Error from "../components/Error";
import { shallow } from "enzyme";

const setUp = () => shallow(<Error />);

describe("Error component tests", () => {
    let component: any;

    beforeEach(() => {
        component = setUp();
    });

    it("Snapshot test", () => {
        expect(component).toMatchSnapshot();
    });
});
