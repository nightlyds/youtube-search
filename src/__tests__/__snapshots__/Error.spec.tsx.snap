// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error component tests Snapshot test 1`] = `
<div
  className="error"
>
  <span
    className="error-message"
  >
    You caught the error :(
  </span>
</div>
`;
